yellowbrick/base.py

```diff
@@ -407,3 +408,119 @@ def predict(self, X, y):
+# anywhere and would be redundant to this new class. Remove that one?
+class MultipleVisualizer(Visualizer):
+    """
+    Used as a base class for visualizers that use subplots.
+
+    Parameters
+    ----------
+    visualizers : A list of instantiated visualizers
+
+    nrows: integer, default: None
+        The number of rows desired, if you would like a fixed number of rows.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify nrows, there will be enough columns created to fit all the 
+        visualizers specified in the visualizers list. 
+    
+    ncols: integer, default: None
+        The number of columns desired, if you would like a fixed number of columns.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify ncols, there will be enough rows created to fit all the 
+        visualizers specified in the visualizers list. 
+
+    axarr: matplotlib.axarr, default: None.
+        If you want to put the plot onto an existing axarr, specify it here. Otherwise a new
+        one will be created. 
+
+    kwargs : additional keyword arguments, default: None
+        Any additional keyword arguments will be passed on to the fit() method and therefore 
+        passed on to the fit() method of the wrapped estimators, if applicable. Otherwise ignored.
+
+    Examples
+    --------
+    >>> from yellowbrick.base import MultipleVisualizer
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from yellowbrick.classifier import ConfusionMatrix
+    >>> from yellowbrick.classifier import ClassBalance
+    >>> model = LogisticRegression()
+    >>> visualizers = [ClassBalance(model),ConfusionMatrix(model)]
+    >>> mv = MultipleVisualizer(visualizers, ncols=2)
+    >>> mv.fit(X_train, y_train)
+    >>> mv.score(X_test, y_test)
+    >>> mv.poof()
+    """
+    def __init__(self, visualizers = [], nrows = None, ncols = None, axarr = None, **kwargs):
+        self.visualizers = visualizers
+        self.plotcount = len(visualizers)
+        if nrows == None and ncols == None:
+            self.ncols = 1
+            self.nrows = self.plotcount
+        elif ncols == None:
+            self.nrows = nrows
+            self.ncols = math.ceil(self.plotcount / self.nrows)
+        elif nrows == None:
+            self.ncols = ncols
+            self.nrows = math.ceil(self.plotcount / self.ncols)
+        else:
+            raise YellowbrickValueError("You can only specify either nrows or ncols, \
+                the other will be calculated based on the length of the list of visualizers.")
+        
+
+        if axarr == None:
+            fig, axarr = plt.subplots(self.nrows, self.ncols, squeeze = False)
+        
+        self.axarr = axarr
+
+        idx = 0
+        for row in range(self.nrows):
+            for col in range(self.ncols):
+                try:
+                    self.visualizers[idx].ax = self.axarr[row, col]
```

Comment:
I very much like assigning the visualizer ax at init. 

If you wanted (and this is not necessary) you could do `self.visualizers[idx]._ax = self.axarr[row, col]` to bypass the `ax` setter; right now the setter does nothing but set `_ax`, so it would not be functionally different but does give readers of the code a sense of the authority of this class.

Reply:
I'll skip for now - I'd be worried about future updates to the base .ax property wanting to do more than set _ax.